runtime/doc/test_urls.vim

```diff
@@ -0,0 +1,68 @@
+" Test for URLs in help documents.
+"
+" Opens a new window with all found URLS followed by return code from curl
+" (anything other than 0 means unreachable)
+"
+" Written by Christian Brabandt.
+
+func Test_check_URLs()
+  if has("win32")
+    echoerr "Doesn't work on MS-Windows"
```

Comment:
`/dev/null` might be part of the problem, but also `TestURL` calls system() which invokes a shell around `curl`, and I think the exit code of cmd.exe might not be reliable. Of course, with Nvim `system()` can be called with a list to avoid invoking a shell.